### ISSUE TITLE
strawberry perl 5.30.1.1 doesn't have  .../c/bin/gsl-config.bat

### ISSUE THREAD

**Issue by aero** [StrawberryPerl/Perl-Dist-Strawberry#13]
Hi
I got the follwing error when I tried to build PDL-2.020 on strawberry perl 5.30.1.1  64bit and 32bit.

```
.
.
Loaded ExtUtils::F77 version 1.24
ExtUtils::F77: Unable to guess and/or validate system/compiler configuration
ExtUtils::F77: Will try system=MinGW Compiler=GFortran
(ExtUtils Version 1.24)

   Building PDL::Slatec. Turn off WITH_SLATEC if there are any problems

'gsl-config'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
배치 파일이 아닙니다.
        no GSL link info (libgsl probably not available)
.
.
```
So gsl related pdl functions are not compiled.

And strawberry perl  5.30.1.1 32bit has another OpenGL related problem.
http://www.cpantesters.org/cpan/report/2790a328-6c85-1014-8029-63bbf30302b6
```
.
.
g++.exe OpenGLQ.def -o ..\..\..\blib\arch\auto\PDL\Graphics\OpenGLQ\OpenGLQ.xs.dll -mdll -s -L"C:\strawberry-perl-5.30-32bit\perl\lib\CORE" -L"C:\strawberry-perl-5.30-32bit\c\lib"    "C:\strawberry-perl-5.30-32bit\perl\lib\CORE\libperl530.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libm.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libmoldname.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libkernel32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libuser32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libgdi32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libwinspool.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libcomdlg32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libadvapi32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libshell32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libole32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\liboleaut32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libnetapi32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libuuid.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libws2_32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libmpr.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libwinmm.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libversion.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libodbc32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libodbccp32.a" "C:\strawberry-perl-5.30-32bit\c\i686-w64-mingw32\lib\libcomctl32.a" -Wl,--enable-auto-image-base
C:/strawberry-perl-5.30-32bit/c/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot export boot_PDL__Graphics__OpenGLQ: symbol not defined
collect2.exe: error: ld returned 1 exit status
gmake[3]: *** [Makefile:466: ..\..\..\blib\arch\auto\PDL\Graphics\OpenGLQ\OpenGLQ.xs.dll] Error 1
.
.
```

**Comment by kmx** [comment]
Ad missing `gsl-config.bat` - it is a bug

Ad. OpenGL issue - at the first sight I have no idea what is going on, please open a separate ticket

**Comment by kmx** [comment]
fixed in 5.30.2.1
